packages/app/obojobo-express/server/routes/api/media.js

```diff
@@ -59,4 +59,21 @@ router
 			.catch(next)
 	})
 
+// Get list of media a thumbnail
+// mounted as /api/media/many
+router
+	.route('/many/')
+	.get([requireCurrentUser])
+	.get((req, res, next) => {
+		const start = parseInt(req.query.start, 10)
```

Comment:
We need to implement some default/min/max values here and make sure start and count are positive ints.

We could use `express-validator` to make sure start and count are sent and are